pipelines/models/bike-rentals-auto-ml/requirements.txt

```diff
@@ -108,6 +108,7 @@ markupsafe==2.1.2
 matplotlib==3.7.1
 matplotlib-inline==0.1.6
 ml-wrappers==0.4.7
+mlflow==2.14.3
 mlflow-skinny==2.2.2
```

Comment:
if we are now including the mlflow dependancy, shouldn't we get rid of the mlflow-skinny dependancy?